dev-libs/libusb/libusb-1.0.23.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="A library that provides generic access to USB devices"
+DESCRIPTION="libusb is meant to facilitate the communication with USB hardware.
+
+- portable: Using a single cross-platform API, it provides access to USB \
+devices on Haiku, Linux, OS X, Android, Windows, OpenBSD, etc.
+- user-mode: No special privilege or elevation is required for the \
+application to communicate with a device.
+- version-agnostic: All versions of the USB protocol, from 1.0 to 3.1 \
+(latest), are supported."
+HOMEPAGE="https://libusb.info/"
+COPYRIGHT="2001 Johannes Erdfelt
+	2007-2009 Daniel Drake
+	2010-2012 Peter Stuge
+	2008-2016 Nathan Hjelm
+	2009-2013 Pete Batard
+	2009-2013 Ludovic Rousseau
+	2010-2012 Michael Plante
+	2011-2013 Hans de Goede
+	2012-2013 Martin Pieuchot
+	2012-2013 Toby Gray
+	2013-2018 Chris Dickens"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/libusb/libusb/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="02620708c4eea7e736240a623b0b156650c39bfa93a14bcfa5f3e05270313eba"
+SOURCE_FILENAME="libusb-$portVersion.tar.gz"
+PATCHES="libusb-$portVersion.patchset"
+
+ARCHITECTURES="x86 x86_64 !x86_gcc2"
```

Comment:
Suggested change (style):
```diff
-ARCHITECTURES="x86 x86_64 !x86_gcc2"
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
```
There is no x86 (primary architecture, so we can't check it there) :)

Reply:
Done